openghg/store/_footprints.py

```diff
@@ -301,6 +301,7 @@ def read_file(
             species = "inert"
         else:
             species = clean_string(species)
+            species = synonyms(species)
```

Comment:
Does this do the right thing if `species="inert"` i.e. leave it unchanged? I assume if the tests pass it does but worth checking that explicitly.